devops/sources_backup.rst

```diff
@@ -0,0 +1,152 @@
+.. _conan_backup_sources:
+
+Backing up third-party sources with Conan
+=========================================
+
+For recipes and build scripts for open source, publicly available libraries,
+it is common practice to download the sources from a canonical source, like Github releases, or project download web pages.
+Keeping a record of the origin of these files is useful for traceability purposes, however,
+it is often not guaranteed that the files will be available in the long term,
+and a user in the future building the same recipe from source may encounter a problem.
+Conan can thus can be configured to transparently retrieve sources from a configured mirror,
+without modifying the recipes or `conandata.yml`.
+Additionally, these sources can be transparently uploaded alongside the packages via :command:`conan upload`.
+
+The *sources backup* feature is intended for storing the downloaded recipe sources in a file server in your own infrastructure,
+allowing future reproducibility of your builds even in the case where the original download URLs are no longer accessible.
+
+The backup is triggered for calls to the :ref:`download<conan_tools_files_get>` and :ref:`get<conan_tools_files_get>` methods
+when a ``sha256`` file signature is provided.
+
+
+.. _backup_sources_config:
+
+Configuration overview
+----------------------
+
+This feature is controlled by a few :ref:`global.conf<reference_config_files_global_conf>` items:
+
+* ``core.sources:download_cache``: Local path to store the sources backups to.
+  *If not set, the default Conan home cache path will be used.*
+* ``core.sources:download_urls``: Ordered list of URLs that Conan will try to download the sources from,
+  where ``origin`` represents the original URL.
```

Comment:
Maybe I would make a mention there that the original url is the one in the _conandata.yml_ just to clarify, because it's a long paragraph that can end up being a bit confusing.